### DISCUSSION TITLE
16 Januar Agenda

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#110] [Weekly Meeting Agenda]
## Protokoll
- Issues aus den User Stories -> #93  (Marvin, Jere & Yannis)
- Component Diagram -> #40 (Yannis)
- 3 neue Label -> Phase 0-2 
    - Phase 0 -> können wir jetzt anfangen
    - Phase 1 -> kann direkt nach 0 umgesetzt werden 
    - Phase 2 -> wird im späteren verlauf bearbeitet
  
### Kritik ERM
- Engement -> Rolls 
   
### Fragen die wir beantwortet haben
- Optional #18 
- Optional #146 
- #132 & #133 optional 
- #101 & #104 optional 
- Machen wir #95 
- Sinnvoll oder zu viel Arbeit: Eine Allgemeine Suche nach Personen in der Schule? Machen wir aber Optional
- Sollen das Hinzufügen von z.B. Schülern, Terminen etc. mit einem Overlay oder einer extra Seite erfolgen? -> Overlay

### Verschoben 
- Kurzer Recap (Welche Aufgaben sind erledigt welche stehen noch an) 
- Aktueller Stand besprechen (Wie liegen wir in der Zeit)
- Vorgehen für die Nächsten 3 Monate besprechen (Wie gehen wir jetzt in der Praxisphase vor)

- Sollen wir einen neuen Termin für die Meetings machen?

## Agenda
Erstes Meeting nach Pause 
- Kurzer Recap (Welche Aufgaben sind erledigt welche stehen noch an) 
- Aktueller Stand besprechen (Wie liegen wir in der Zeit)
- Vorgehen für die Nächsten 3 Monate besprechen (Wie gehen wir jetzt in der Praxisphase vor)

- Issues aus den User Stories erstellen 
- Component Diagramm 

### Extra Fragen
(Nicht alle Fragen werden direkt in diesem Meeting beantwortet, da ein Teil der Gruppe noch im Urlaub ist)
- Sollen wir einen neuen Termin für die Meetings machen?
-  Ist nicht mehr teil des Projektes #18 
- Wie sollen die Benachrichtigungen zu Timed Events entstehen (1 Tag vor z.B. Klassenfahrt Benachrichtigung)  #146
                -  Was für Benachrichtigungen gibt es  
- #132 #133, #101 #104 und #95 (und damit verbundene Funktionen) optional oder essential?

### Extra Features 
- Sinnvoll oder zu viel Arbeit: Eine Allgemeine Suche nach Personen in der Schule?

**Comment by Schmarvinius** [comment]
Was für Benachrichtigungen gibt es
Wie sollen die Benachrichtigungen zu Timed Events entstehen (1 Tag vor z.B. Klassenfahrt Benachrichtigung)
#146

**Reply by Kronprinz03** [reply]
Meine Idee wäre auf spontan, dass naja Benachrichtigungen trifft es vllt nicht so ganz konkret eher halt nochmal ne Ansicht mit allem was kurz bevor steht

**Comment by Schmarvinius** [comment]
#132 #133, #101 #104 und #95 (und damit verbundene Funktionen) optional oder essential?

Terminkalender Funktionen:
- #146
- #99 #100 (damit ist gemeint einerseits z.B. Klassenfahrten aber auch Klassenarbeiten und co.)

**Reply by Kronprinz03** [reply]
#132 & #133 optional 
#101 & #104 optional 
#95 Wäre schon wichtig allerdings keine hohe Priorität 

Würde ich jetzt spontan sagen allerdings besprechen wir dann nochmal im Meeting

**Comment by Schmarvinius** [comment]
Sinnvoll oder zu viel Arbeit: Eine Allgemeine Suche nach Personen in der Schule?

**Reply by Kronprinz03** [reply]
Viel Arbeit aber kann ein cooles Feature sein können wir als optional machen
